src/CSS/index.css

```diff
@@ -88,6 +99,25 @@ body {
 input:checked + label svg.sun {
   fill: #7e7e7e;
 }
+
 input:checked + label svg.moon {
   fill: black;
 }
+
+@media screen and (max-width: 490px) {
+  .block {
+    font-size: 13px;
+  }
+}
+
+@media screen and (max-width: 390px) {
+  .block {
+    font-size: 11px;
+  }
+}
+
+@media screen and (max-width: 330px) {
+  .block {
+    font-size: 9px;
```

Comment:
could you add using Tailwind CSS? we are using Tailwind CSS

Reply:
> could you add using Tailwind CSS? we are using Tailwind CSS

Yeah sure here you go :)

Reply:
Yes I have made changes